vignettes/piggyback.Rmd

```diff
@@ -145,6 +147,10 @@ pb_download_url(repo = "cboettig/piggyback-tests",
 [4] "https://github.com/cboettig/piggyback-tests/releases/download/v0.0.1/iris2.tsv.gz" 
 ```
 
+Note that these will not work if authentication is required to download them
+
+## Reading data
+
```

Comment:
in progress, will finish later this weekend hopefully. 

Will try to add examples of reading URLs above, and then will demonstrate pb_read/write below including how to pass it other read_functions.